src/metrics/chaoss.ts

```diff
@@ -520,6 +520,53 @@ FORMAT JSONCompact`;
   });
 };
 
+export const chaossChangeRequestsAcceptedRatio = async (config:QueryConfig) => {
+  config = getMergedConfig(config);
+  const whereClauses: string[] = ["type = 'PullRequestEvent' AND action = 'closed' "];
+  const repoWhereClause = await getRepoWhereClauseForClickhouse(config);
+  if (repoWhereClause) whereClauses.push(repoWhereClause);
+  whereClauses.push(getTimeRangeWhereClauseForClickhouse(config));
+  const sql = `
+SELECT
+  id,
+  argMax(name, time) AS name,
+  ${getGroupArrayInsertAtClauseForClickhouse(config, { key: 'change_requests_accepted_ratio', value: 'ratio' })},
+  ${getGroupArrayInsertAtClauseForClickhouse(config, { key: 'change_requests_accepted', value: 'acount' })},
+  ${getGroupArrayInsertAtClauseForClickhouse(config, { key: 'change_requests_declined', value: 'dcount' })}
+
+FROM
+(
+  SELECT
+    ${getGroupTimeAndIdClauseForClickhouse(config, 'repo')},
+    COUNT() AS count,
+    countIf(pull_merged = 1) AS acount,
+    countIf(pull_merged = 0) AS dcount,
+    round(acount/count,2) AS ratio
+  FROM github_log.events
+  WHERE ${whereClauses.join(' AND ')}
+  GROUP BY id, time
+  ${config.limitOption === 'each' && config.limit > 0 ? 
+    `${config.order ? `ORDER BY ratio ${config.order}` : ''} LIMIT ${config.limit} BY time` :
+    ''}
+)
+GROUP BY id
+${config.order ? `ORDER BY change_requests_accepted_ratio[-1] ${config.order}` : ''}
+${config.limitOption === 'all' && config.limit > 0 ? `LIMIT ${config.limit}` : ''}
+FORMAT JSONCompact`;
+
+  const result: any = await clickhouse.query(sql);
+  return result.map(row => {
+    const [id, name, ratio,acount,dcount] = row;
```

Comment:
You missed blank space before `account` and `dcount`.

And I think you should use `accept_count`, `declined_count` and `accept_ratio` rather than `acount`, `dcount` and `ratio`, since OpenDigger data will release to OSS, and we should try our best to make sure users can understand the data without documentation.